docs/Add-a-module.md

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
Imho the file name should be either "How-to-add-a-module.md" or "Adding-a-module"